bot/src/alert.js

```diff
@@ -1,37 +1,19 @@
 const { listSubscriptions } = require('./db');
 const { TELEGRAM_TOKEN } = require('./environment');
+const axios = require('axios');
 const https = require('https');
 
-function alert(name, message) {
-    listSubscriptions(name).then((subss) => {
-        let sendMessagePath = `bot${TELEGRAM_TOKEN}/sendMessage`;
-
-        subss.forEach((subs) => {
-            let req = https.request({
-                host: "https://api.telegram.org/",
-                path: sendMessagePath,
-                method: "POST",
-                headers: {
-                    "Content-Type": "application/json"
-                }
-            }, (res) => {
-                if (res.statusCode == 200) {
-                    console.log(`Sent message to '${subs}'`);
-                } else {
-                    console.log(`Couldn't send message to '${subs}'`);
-                }
-            });
-
-            req.write(JSON.stringify({
-                chatId: subs,
-                text: message
-            }));
-
-            req.end();
-
-        });
+async function alert(name, message) {
+    let subss = await listSubscriptions(name); 
+    let sendMessagePath = `https://api.telegram.org/bot${TELEGRAM_TOKEN}/sendMessage`;
+    let promisses = subss.map((chatId) => {
+        return axios.post(sendMessagePath,
+            {
+                chat_id: chatId,
+                text: `${message}`
+            })
```

Comment:
@gutorc92, I see you deleted the `console.log` when a message couldn't be sent. Since this operation runs without previous user interaction, we cannot let him know that the message could not be sent. Hence, we have to find a way to know if everything is running smoothly. I think a big improvement would be the addition of a cloud-native log library here.